dependencies.yaml

```diff
@@ -282,6 +281,9 @@ dependencies:
               - cuda-nvrtc-dev
               - cuda-nvtx-dev
               - libcurand-dev
+              # TODO: Remove after https://github.com/rapidsai/cudf/pull/14292 updates
+              # cudf_kafka to use scikit-build
+              - cuda-gdb
```

Comment:
Same as above -- we can remove this since we are specifying `CUDA_HOME` in the cudf_kafka build script.